src/Command/DumpCommand.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Phoenix\Command;
+
+use Phoenix\Command\AbstractCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DumpCommand extends AbstractCommand
+{
+    protected function configure()
```

Comment:
Mark as executed doesn't make sense. It will be marked only on system where it was created...